### ISSUE TITLE
Unable to delete objects in a folder with empty name ("")

### ISSUE THREAD

**Issue by defel** [rebuy-de/aws-nuke#65]
Hi,

first: I love this tool. I found it today and it protected me very effective to delete unintentionally some roles, and it nuked all experimental created resources. Very good job! 

I found a small bug:

It is possible to create folders in S3 which have an empty folder-name. `aws-nuke` is trying to delete the files in this folder again and again. 

Example output:
```
eu-central-1 - S3Object - 's3://xyz-dev//eu-central-1/dev/xyz-infrastructure-ecs-2017-12-08-09-22-47-132549Z.json' - waiting
```
As you can see between the bucket-name "xyz-dev" and 2nd-level folder-name "eu-central-1" there is a empty folder. 

I had to delete this folder manually and it works now.

Thanks again!

**Comment by svenwltr** [comment]
I am very sorry for the late response. This ticket got lost somethow.

The issue is somewhere in the AWS SDK. *aws-nuke* is passing the correct S3 Object key for the SDK, but it does a request without the second `/`. I try to figure out how to resolve this problem.

**Comment by svenwltr** [comment]
I created an error report in the SDK: https://github.com/aws/aws-sdk-go/issues/1853.

**Comment by svenwltr** [comment]
Will be fixes in #163.
